deployment/cfn/data_plane.py

```diff
@@ -299,10 +299,10 @@ def create_rds_instance(self):
             DBParameterGroupName=Ref(rds_parameter_group),
             DBSubnetGroupName=Ref(rds_subnet_group),
             Engine='postgres',
-            EngineVersion='9.4.1',
+            EngineVersion='9.4.20',
             MasterUsername=Ref(self.rds_username),
             MasterUserPassword=Ref(self.rds_password),
-            MultiAZ=True,
+            MultiAZ=Ref(self.rds_multi_az),
```

Comment:
I parameterized this to reconcile the DataPlane drift in staging, which had this set to `False`.